lib/main.dart

```diff
@@ -68,33 +69,68 @@ class _RandomWordsState extends State<RandomWords> {
     );
   }
 
+  void _sortSaved() {
+    _sorted.clear();
+
+    List _listSorted = _saved.toList();
+    _listSorted.sort((a, b) {
+      return a.first.toLowerCase().compareTo(b.first.toLowerCase());
+    });
+
+    _listSorted.forEach((wordPair) => _sorted.add(wordPair));
+    _pushSaved();
+  }
+
   void _pushSaved() {
     Navigator.of(context).push(
-      // NEW lines from here...
       MaterialPageRoute<void>(
         builder: (BuildContext context) {
-          final tiles = _saved.map(
-            (WordPair pair) {
-              return ListTile(
-                title: Text(
-                  pair.asPascalCase,
-                  style: _biggerFont,
-                ),
-              );
-            },
-          );
+          final tiles = _sorted.isNotEmpty
+              ? _sorted.map(
+                  (WordPair pair) {
```

Comment:
use lambda funcation to save your selector to a separate variable, and then reuse it in both clauses:
```
final tiles = _sorted.isNotEmpty ? _sorted.map(selector) : _saved.map(selector)
```